packages/components/src/menu/tests/chromatic/Menu.stories.tsx

```diff
@@ -0,0 +1,330 @@
+import { DeleteIcon, EditIcon, KebabIcon, SparklesIcon } from "@hopper-ui/icons";
+import { hopperParameters } from "@hopper-ui/storybook-addon";
+import { Div } from "@hopper-ui/styled-system";
+import type { Meta, StoryObj } from "@storybook/react";
+import { userEvent } from "@storybook/test";
+
+import { Avatar } from "../../../avatar/index.ts";
+import { Button } from "../../../buttons/index.ts";
+import { Divider } from "../../../divider/index.ts";
+import { Header } from "../../../header/index.ts";
+import { IconList } from "../../../icon-list/index.ts";
+import { Text } from "../../../typography/index.ts";
+import { Menu, MenuItem, MenuSection, MenuTrigger, SubmenuTrigger } from "../../src/index.ts";
+import { CombinedMenu } from "../../src/Menu.tsx";
+
+const meta = {
+    title: "Components/Menu",
+    component: CombinedMenu,
+    decorators: [
+        Story => (
+            <Div UNSAFE_height="300px">
+                <Story />
+            </Div>
+        )
+    ],
+    args: {
+        isOpen: true
+    }
+} satisfies Meta<typeof CombinedMenu>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default = {
+    render: args => (
+        <MenuTrigger {...args}>
+            <Button variant="secondary" aria-label="Actions for selected resource">
+                <KebabIcon />
+            </Button>
+            <Menu {...args}>
+                <MenuItem>Favorite</MenuItem>
+                <MenuItem>Edit</MenuItem>
+                <MenuItem>Delete</MenuItem>
+            </Menu>
+        </MenuTrigger>
+    )
+} satisfies Story;
+
+export const Sections = {
```

Comment:
can you add a tests with a submenu and a selection mode. To make sure that the check doesn't appear and only the > is present